code/obj/critter/misc.dm

```diff
@@ -484,9 +484,23 @@
 
 
 	CritterAttack(mob/M)
-		take_bleeding_damage(M, M, rand(3,6), DAMAGE_STAB, 1)
-		M.visible_message("<span class='combat'><B>[src]</B> snips [src.target] with its pincers!</span>")
-		playsound(src.loc, "sound/items/Wirecutter.ogg", 50, 0)
+		if(!ON_COOLDOWN(src, "scorpion_ability", 15 SECONDS))
+			if(prob(50))
+				M.visible_message("<span class='combat'><B>[src]</B> stings [src.target]!</span>")
+				M.reagents?.add_reagent("neurotoxin", 15)
```

Comment:
Toxin by itself seems enough.

Reply:
they've had neuro since the beggining, i just added the code here so they can't get stuck on the pincer attacks on the rare cases the person they've attacked hasn't moved away, and changed the delay to 15 seconds to make it overall weaker